internal/distro/rhel8/imagetype.go

```diff
@@ -38,6 +38,11 @@ const (
 
 	// blueprint package set name
 	blueprintPkgsKey = "blueprint"
+
+	// hardcode the datastream since this is provided by the scap-security-guide rpm
+	// and is not expected to change.
+	centosDatastream = "/usr/share/xml/scap/ssg/content/ssg-centos8-ds.xml"
+	rhelDatastream   = "/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml"
```

Comment:
again, I would like to see these names prefixed with `oscap` to make it obvious what they are for...